### ISSUE TITLE
Add link rel support for structural units

### ISSUE THREAD

**Issue by waldoj** [statedecoded/statedecoded#119]
Next, prev, and up link rel tags should be supported within structural units, along with keyboard navigation (as is used for individual laws).
